src/caret_analyze/plot/bokeh/histogram.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from bokeh.plotting import figure, show
+
+from caret_analyze.exceptions import UnsupportedTypeError
+
+from caret_analyze.record import ResponseTime
+
+from .histogram_interface import HistPlot
+
+from .plot_util import PlotColorSelector
+
+
+class ResponseTimePlot(HistPlot):
+
+    def __init__(
+        self,
+        target,
+        case='default'
```

Comment:
`default` is little vague.
How about `best-to-worst`?

There may be more better words.